bin/copy_robots_file

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+
+require 'login_gov/hostdata'
+
+if LoginGov::Hostdata.in_datacenter? && LoginGov::Hostdata.env != 'prod'
```

Comment:
Slight preference for using `LoginGov::Hostdata.domain != 'login.gov'` instead of `LoginGov::Hostdata.env != 'prod'`, which would allow us to test that this works in staging before it goes to production.